docs/requirements.md

```diff
@@ -0,0 +1,129 @@
+# Business requirements
+
+> TODO
+
+# User requirements
+
+## Use cases
+
+### Custom Consumer Chain
+
+> TODO e.g., Quicksilver
+
+### Contract Consumer Chains
+
+> TODO e.g., TBA 
+
+### Governance-Gated CosmWasm Consumer Chains
+
+> TODO e.g., Lido
+
+# Functional requirements
+
+## Assumptions
+
+1. Interchain Security packets will (almost) never timeout.
+
+## Features
+
+### 1 - Bootstrapping
+
+| ID  | Description | Verification | Status | Release |
```

Comment:
I think it would be better if we used 'correctness' or 'quality assurance' instead of verification, as verification is a bit loaded.